telemetry/otel.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package telemetry initializes the OpenTelemetry SDK for library users. It
```

Comment:
Have you looked at https://cloud.google.com/stackdriver/docs/instrumentation/setup/go when making this?

Reply:
Yea I looked at many examples - they're more or less the same format. But ended up taking with what worked for Tapestry and the collector-contrib example app in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/examples/demo/server/main.go.